### DISCUSSION TITLE
Ayuda, necesito orientacion

### DISCUSSION THREAD

**Discussion by AdoRockdriguez** [Launch-X-Latam/MisionFrontEnd#464] [Team Posts]
Hola. 
Por favor necesito orientacion, sobre como y a donde subir las practicas, por trabajo me atrase en los lives y apanas estoy comenzando CSS. Gracias

**Comment by AlvaroFierro** [comment]
Hola, te puede servir la discusión #247

**Comment by AbnerHub** [comment]
Disculpa ¿Cómo encuentro la discusión ?

**Comment by gfrodriguez** [comment]
> [247](https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/247)

**Comment by AlvaroFierro** [comment]
Una disculpa, te respondí por GitHub mobile y según si estaba el link 😅 pero gracias @gfrodriguez

**Comment by jorgesniper** [comment]
https://youtu.be/L_lWQZNhN7w  a mi me sirvió mucho este video espero les sirva a ustedes tambien

**Comment by AbnerHub** [comment]
Muhcas gracas @gfrodriguez
